src/views/portals/YetiView.vue

```diff
@@ -595,6 +778,59 @@
         result += '&username=' + this.$user.forumUsername;
 
         return result;
+      },
+
+      onMapMoveEnd() {
+        this.setVisibleMountains();
+      },
+
+      setVisibleMountains() {
+        const mapExtent = this.$refs.map.getExtent('EPSG:4326');
+        // clone this.mountains first, with no reference
+        this.visibleMountains = Object.assign({}, this.mountains);
+        // then filter if polygon isn’t in view
+        for (const massif in this.visibleMountains) {
+          this.visibleMountains[massif] = this.visibleMountains[massif].filter(mountain => {
+            const polygon = mountain.geometry;
+            return polygon.intersectsExtent(mapExtent);
+          });
+          // unset massif if empty
+          if (this.visibleMountains[massif].length === 0) {
+            delete this.visibleMountains[massif];
+          }
+        }
+      },
+
+      sortMountainsByMassif() {
```

Comment:
I do not fully understand the sorting achieved? It is based on the moutain field, alphabetically?

Reply:
It's in order to sort mountains (Chablais, Aravis, etc.) by massifs (Alpes du Nord, Alpes du Sud, etc.) as seen in UI.
Do you mean that mountains are not sorted alphabetically in each massifs?

Reply:
Added alphabetically sorting